src/microbe_stage/organelle_components/LysosomeComponent.cs

```diff
@@ -0,0 +1,61 @@
+﻿using Godot;
+
+public class LysosomeComponent : IOrganelleComponent
+{
+    public void OnAttachToCell(PlacedOrganelle organelle)
+    {
+        var configuration = organelle.Upgrades?.CustomUpgradeData;
+
+        var upgrades = configuration as LysosomeUpgrades;
+
+        var enzyme = configuration == null ? SimulationParameters.Instance.GetEnzyme("lipase") : upgrades!.Enzyme;
```

Comment:
```suggestion
        var enzyme = upgrades == null ? SimulationParameters.Instance.GetEnzyme("lipase") : upgrades.Enzyme;
```
I think that will satisfy the nullability checker